cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	unspecifiedSourceError   = "You need to specify the source for the add-on."
+	unspecifiedSourceError   = "You must specify the source file path of the add-on."
```

Comment:
We need to provide path to the directory which contains the addon file. So we need to reword this e.g. `You must specify the source directory path of the add-on` or something similar.